DOCSUP-1249 fixedRandomString description was added (#137)
BayoNet/ClickHouse: docs/en/sql-reference/functions/other-functions.md

```

[Original article](https://clickhouse.tech/docs/en/query_language/functions/other_functions/) <!--hide-->

## randomFixedString {#randomfixedstring}

> Accepts natural number of the length variable. Returnes generated random string with fixed length.

Reviewer: ```suggestion
Generates a random string of a specified length.
```

**Syntax**

``` sql
randomFixedString(length);
```

**Parameters**

> -   `length` — Length to put in the function number of symbols we need to generate. Length is a natural number. [UInt64](../../sql-reference/data-types/int-uint.md).

Reviewer: ```suggestion
-   `length` — Required number of symbols in the resulting string. [UInt64](../../sql-reference/data-types/int-uint.md).
```

**Returned value(s)**

> -   String contains generated random symbols.

Reviewer: ```suggestion
-   String of random symbols.
```
